### DISCUSSION TITLE
Add Kees1958 most US+EU ad and tracking servers to Multipurpose blocklist section (it is better than Peter Low's for people living in the EU)

### DISCUSSION THREAD

**Discussion by LennyFox** [uBlockOrigin/uAssets#13158] [Ideas]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

Everywhere

### Describe the issue

A multi purpose list is missing. I see only 
a) Dan Pollock's host file
b) Peter Low's Ad and Tracking Servers

but I can't see
c) Kees1958 EU_US+most_used_ad_and_tracking_networks

Here is the link to the raw file: https://raw.githubusercontent.com/Kees1958/W3C_annual_most_used_survey_blocklist/master/EU_US%2Bmost_used_ad_and_tracking_networks

### uBlock Origin version

uBlock Origin 1.42.4

### Browser name and version

Edge 101.0.1210.39

### Settings

None

### Notes

Could not find a request in uBlock Assets to add this multi purpose list

**Comment by mapx-** [comment]
You can add it as a custom list

**Reply by LennyFox** [reply]
First thanks @mapx for moving it to the correct section. Secondly, yes I have done that. I thought it might be an idea to add this list to the multi-purpose since Gorhill removed a few inactive lists. For me (living in EU) the Kees1958 performs better than Peter Low's. I like that Kees1958 blocks third-party only and uses * as entity to catch all TLD/Country codes.
